helm/cluster-vsphere/templates/helmreleases/cloud-provider-vsphere-helmrelease.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   chart:
     spec:
       chart: cloud-provider-vsphere
-      version: 1.5.0
+      version: 1.5.1
```

Comment:
because `1.5.1` has the necessary CNP [shipped](https://github.com/giantswarm/cloud-provider-vsphere-app/pull/65)